<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobeSettings</class>
 <widget class="QMainWindow" name="LobeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>348</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Lobe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="lobeName">
      <property name="text">
       <string>Lobe Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="SetColor">
      <property name="text">
       <string>Set Color</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="ColorWidget" name="ColorDisplay" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="CellsPerColumn">
      <property name="text">
       <string>Cells Per Column</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="depth">
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="depthVal">
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="LobeDynamics">
      <property name="title">
       <string>Lobe Dynamics</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="DesiredActivity">
         <property name="text">
          <string>Desired Activity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="activity">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="activityVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="MaximumBoost">
         <property name="text">
          <string>Maximum Boost</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="max_boost">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="boostVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="DutyCycle">
         <property name="text">
          <string>Duty Cycle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="duty_cycle">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="dutyVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Period">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="period">
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>30000</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="periodVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QGroupBox" name="NeuronDynamics">
      <property name="title">
       <string>Neuron Dynamics</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Threshold">
         <property name="text">
          <string>Threshold</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="threshold">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="threshVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="RelaxState">
         <property name="text">
          <string>Relax State</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="relax_state">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="stateVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="RelaxMemory">
         <property name="text">
          <string>Relax Memory</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="relax_memory">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="prevVal">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CloseButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>LobeSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>depthVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threshVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duty_cycle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dutyVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>activityVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_boost</sender>
   <signal>valueChanged(int)</signal>
   <receiver>boostVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>period</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodVal</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
